### PR TITLE
parser: support `secondary_load` and `secondary_unload`

### LINKED ISSUE #444: Fix SECONDARY_LOAD/SECONDARY_UNLOAD syntax
Index Issue: pingcap/tidb#11486

These are bad (incompatible) SQL cases:

```sql
ALTER TABLE d_n.t_n SECONDARY_LOAD
ALTER TABLE t_n LOCK = DEFAULT , SECONDARY_LOAD
ALTER TABLE d_n.t_n ALGORITHM = DEFAULT , SECONDARY_LOAD
ALTER TABLE d_n.t_n SECONDARY_UNLOAD
```

Claim this subtask based on the comments below this issue.

### PR DESCRIPTION
**Pull request by spongedu**
### What problem does this PR solve? 
Fix #444 , support `SECONDARY_LOAD` and `SECONDARY_UNLOAD` option in `ALTER TABLE` syntax

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

Code changes

N / A 

Side effects

N / A 

Related changes

N / A

### CODE REVIEW

File: ast/ddl.go [hunk 1]
```diff
@@ -1838,6 +1838,8 @@ const (
 	AlterTableRemovePartitioning
 	AlterTableWithValidation
 	AlterTableWithoutValidation
+	AlterTableSecondaryLoad
+	AlterTableSecondaryUnLoad
```
Comment (kennytm) [inline ast/ddl.go]: ... but I'd appreciate if you spell it as 

```suggestion
	AlterTableSecondaryUnload
```

Comment (spongedu) [reply ast/ddl.go]: ok

**Comment by leoppro** [comment]
@spongedu  resolve conflicts please

**Comment by leoppro** [comment]
@spongedu could you resolve conflicts after we merged #505

**Pull request merged**